starknet/src/utils/types.cairo

```diff
@@ -120,6 +120,23 @@ struct Strategy {
     params: Array<felt252>,
 }
 
+trait StrategyTrait {
```

Comment:
do you think this should be defined in the test folder somewhere? as the trait would only ever be used in tests.

Reply:
bbf7ca6a0345444477f6bbc6adb8e6ea63f5f731